File: KITTY/KURS_1/JAVA/DAY_31_1_ListFragment/app/src/main/java/com/example/velic/day_31_1_listfragment/SingleListFragment_1.java
package com.example.velic.day_31_1_listfragment;


import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListAdapter;

import androidx.fragment.app.ListFragment;

public class SingleListFragment_1 extends ListFragment {
    // определяем массив типа String
    final String[] catNames = new String[]{"Рыжик", "Барсик", "Мурзик",
            "Мурка", "Васька", "Томасина", "Кристина", "Пушок", "Дымка",
            "Кузя", "Китти", "Масяня", "Симба"};

    @Override
    public void onActivityCreated(Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        ListAdapter adapter = new ArrayAdapter<>(getActivity(),
                android.R.layout.simple_list_item_1, catNames);
        setListAdapter(adapter);
    }
}

File: KITTY/KURS_1/KOTLIN/DAY_31_1_LIST_FRAGMENT_kt/app/src/main/java/com/example/day_31_1_list_fragment_k/SingleListFragment_1.kt
package com.example.day_31_1_list_fragment_k


import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.ListAdapter

import androidx.fragment.app.ListFragment

class SingleListFragment_1 : ListFragment() {
    // определяем массив типа String
    internal val catNames = arrayOf("Рыжик", "Барсик", "Мурзик", "Мурка", "Васька", "Томасина", "Кристина", "Пушок", "Дымка", "Кузя", "Китти", "Масяня", "Симба")

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)

        val adapter = ArrayAdapter(activity!!,
                android.R.layout.simple_list_item_1, catNames)
        listAdapter = adapter
    }
}
